src/main/kotlin/lotto/domain/WinningStatistics.kt

```diff
@@ -2,20 +2,25 @@ package lotto.domain
 
 import lotto.service.LottoCalculator
 
-class WinningStatistics(val winningStatistics: Map<Int, Int>) {
+class WinningStatistics(val winningStatistics: Map<Rank, Int>) {
     fun calculateTotalPrizeMoney(): Long {
         return winningStatistics
-            .map { (matchCount, count) ->
-                LottoCalculator.calculatePrizeMoney(matchCount) * count
+            .map { (rank, count) ->
+                LottoCalculator.calculatePrizeMoney(rank) * count
```

Comment:
rank 객체에게 메시지를 보내도록 변경해볼 수 있을 것 같아요. 🤔
(네이밍은 임의로 지어봤습니다.  👀)
```suggestion
                rank.multiplyWinningMoney(count)
```

Reply:
오오... operator fun times로 바꿔보겠습니다.